packages/create-gatsby/src/index.ts

```diff
@@ -179,7 +189,7 @@ ${center(c.blueBright.bold.underline(`Welcome to Gatsby!`))}
     format: (value: string): string => c.cyan(value),
   } as any)
 
-  const data = await enquirer.prompt(questions(kebabify(siteName)))
+  const data = await enquirer.prompt(questions(kebabify(siteName), yesFlag))
```

Comment:
Would it be simpler to just pass in the first question to prompt if, rather than adding support for skips to the custom prompts? Something like this:

```suggestion
  const prompts = questions(kebabify(siteName))
  const data = await enquirer.prompt(yesFlag ? [prompts[0]] : prompts))
```